lib/logmatcher.c

```diff
@@ -209,6 +209,7 @@ log_matcher_string_new(const LogMatcherOptions *options)
   self->super.compile = log_matcher_string_compile;
   self->super.match = log_matcher_string_match;
   self->super.replace = log_matcher_string_replace;
+  self->super.prefix = NULL;
```

Comment:
this is not needed if https://github.com/syslog-ng/syslog-ng/pull/3702/files#r650710436 is fixed.